src/wallet/spend.cpp

```diff
@@ -1161,7 +1161,19 @@ static util::Result<CreatedTransactionResult> CreateTransactionInternal(
     if (!select_coins_res) {
         // 'SelectCoins' either returns a specific error message or, if empty, means a general "Insufficient funds".
         const bilingual_str& err = util::ErrorString(select_coins_res);
-        return util::Error{err.empty() ?_("Insufficient funds") : err};
+        if (!err.empty()) return util::Error{err};
+
+        // Check if we have enough balance but cannot cover the fees
+        CAmount available_balance = preset_inputs.total_amount + available_coins.GetTotalAmount();
```

Comment:
Note to reviewers: `preset_inputs.total_amount` will be the sum of the outputs’ effective values if "subtract fee from outputs" is disabled, but will be the nominal amount if SFFO is active. See [`src/wallet/spend.h:164–172`](https://github.com/bitcoin/bitcoin/blob/8cefff322cc6689e50074c93cb9d52412d614d13/src/wallet/spend.h#L164)